packages/data-stores/src/plans/actions.ts

```diff
@@ -3,47 +3,64 @@
  */
 import type { Plan, PlanFeature, FeaturesByType, PlanProduct } from './types';
 
-export const setFeatures = ( features: Record< string, PlanFeature >, locale: string ) => {
-	return {
-		type: 'SET_FEATURES' as const,
-		features,
-		locale,
-	};
+type setFeaturesAction = {
+	type: 'SET_FEATURES';
+	features: Record< string, PlanFeature >;
+	locale: string;
 };
+export const setFeatures = (
+	features: Record< string, PlanFeature >,
+	locale: string
+): setFeaturesAction => ( {
+	type: 'SET_FEATURES' as const,
+	features,
+	locale,
+} );
 
-export const setFeaturesByType = ( featuresByType: Array< FeaturesByType >, locale: string ) => {
-	return {
-		type: 'SET_FEATURES_BY_TYPE' as const,
-		featuresByType,
-		locale,
-	};
+type setFeaturesByTypeAction = {
+	type: 'SET_FEATURES_BY_TYPE';
+	featuresByType: Array< FeaturesByType >;
+	locale: string;
 };
+export const setFeaturesByType = (
+	featuresByType: Array< FeaturesByType >,
+	locale: string
+): setFeaturesByTypeAction => ( {
+	type: 'SET_FEATURES_BY_TYPE' as const,
+	featuresByType,
+	locale,
+} );
 
-export const setPlans = ( plans: Plan[], locale: string ) => {
-	return {
-		type: 'SET_PLANS' as const,
-		plans,
-		locale,
-	};
+type setPlansAction = {
+	type: 'SET_PLANS';
+	plans: Plan[];
+	locale: string;
 };
+export const setPlans = ( plans: Plan[], locale: string ): setPlansAction => ( {
+	type: 'SET_PLANS' as const,
+	plans,
+	locale,
+} );
 
-export const setPlanProducts = ( products: PlanProduct[] ) => {
-	return {
-		type: 'SET_PLAN_PRODUCTS' as const,
-		products,
-	};
+type setPlanProductsAction = {
+	type: 'SET_PLAN_PRODUCTS';
+	products: PlanProduct[];
 };
+export const setPlanProducts = ( products: PlanProduct[] ): setPlanProductsAction => ( {
+	type: 'SET_PLAN_PRODUCTS' as const,
+	products,
+} );
 
-export const resetPlan = () => {
-	return {
-		type: 'RESET_PLAN' as const,
-	};
-};
+type resetPlanAction = { type: 'RESET_PLAN' };
+export const resetPlan = (): resetPlanAction => ( {
+	type: 'RESET_PLAN' as const,
+} );
 
 export type PlanAction = ReturnType<
```

Comment:
[Nit pick]: Doesn't it make more sense to use a plural here since we exporting multiple action types?

Reply:
Probably good to keep it as is, since this type describes what a `PlanAction` type is